autoware_plugin/src/autoware_plugin.cpp

```diff
@@ -42,7 +42,7 @@ namespace autoware_plugin
         plugin_discovery_msg_.available = true;
         plugin_discovery_msg_.activated = false;
         plugin_discovery_msg_.type = cav_msgs::Plugin::STRATEGIC;
-        plugin_discovery_msg_.capability = "waypoint_following/autoware";
```

Comment:
Emm... I am not sure that is a actual defect.The capability here is described as "top-level-feature/implementation" instead of an actual service name. The service name is composed by health_monitor using its config.yaml file.
```# String: Plugin service name prefix
plugin_service_prefix: /guidance/plugin/

# String: Strategic plugin service suffix
strategic_plugin_service_suffix: /plan_maneuver

# String: Tactical plugin service suffix
tactical_plugin_service_suffix: /plan_trajectory